apiary-metastore-listener/README.md

```diff
@@ -83,7 +83,8 @@ The following shows an example JSON message representing an "ADD_PARTITION" even
       "eventType": "ADD_PARTITION",
       "dbName": "some_db",
       "tableName": "some_table",
-      "partition": ["col_1", "col_2", "col_3"]
+      "partitionKeys": {"Column_2":"String", "Column_1":"String", "Column_3":"String"},
```

Comment:
This probably should be 
 "partitionKeys": ["col_1", "col_2", "col_3"],

Reply:
GH can be annoying, I replied to your comment but it's gone in the comment view not here...

Reply:
I think it's easier to read if you format this like we do with the CT SNS documentation, i.e.

```
"partitionKeys": {
    "Column_2": "string", 
    "Column_1": "string", 
    "Column_3": "int"
 },
```
Also, let's make the type names all lower case and have an "int" to demonstrate that we don't just support strings.

Reply:
ok In that case probably we need to update code accordingly.

what I noticed is ADD_PARTITION & DROP_PARTITION events are using same code but README is different.

Reply:
Alright. so, heres what happened. I implemented the PartitionKey as a Map and started updating the README file and then realized that the order of the columns had changed in the JSONObject but because I had already done the code for it, I didn't want to loose the progress and hence I just committed all of it here.

Then I reverted all of those changes to make the PartitionKeys back as an Array and committed it but unfortunately missed reverting the README file which lead to this confusion. 

We have discussed internally and I am going to explore some other JSON library to achieve the ordering of the partition keys tomorrow and will update the code accordingly.